workshopapps/todoassistant.web
# FE-63: feat VA Task Page

Task : Virtual Assistant All Tasks Page with functionality
Task Description: consume the API thats displays all tasks assigned to a Virtual Assiststant 
 
API ENDPOINTS:
/tasks/all/va

VIRTUAL ASSISTANT LOGIN CRED:
email address: 2@a.com
password: secret123 

FIGMA LINK:
https://www.figma.com/file/3f7fQODyHQ0PC7e2skvD8O/Ticked?node-id=4180%3A61304&t=rKTPZz64uv5zeZkg-0

LIVE LINK:
https://ticked.hng.tech/virtual-assistance/tasks

HOW TO TEST:
step 1: login as a virtual assistant with the credentials above
step 2: go to the tasks section on the side navigation(click it)
at this point the api (returns the users associated with the VA) is called,
a table is displayed with the users.



ERRORS YOU MIGHT ENCOUNTER:

Internal server error -- error 500
data returned by api might render in HTML/text instead of application/json
this error only happened on the live site! (backend dev said its a deployment issue)
VIEWS
